### PR TITLE
build/ci-Dockerfile `chmod -R 777 /go/`

### PR DESCRIPTION
**Pull request by kaovilai**
We want to make /go/ writable for all users to resolve [CI permission issue](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/35643/rehearse-35643-pull-ci-openshift-oadp-operator-master-4.9-operator-e2e-aws/1617908146062233600#1:build-log.txt%3A135)

### CODE REVIEW

File: build/ci-Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM quay.io/konveyor/builder:v1.17.2 AS builder
+FROM quay.io/konveyor/builder:v1.18 AS builder
```
Comment (sseago) [inline build/ci-Dockerfile:2]: Actually, we should probably just use `quay.io/konveyor/builder` and we'll get 1.19

Comment (sseago) [reply build/ci-Dockerfile:2]: (but if this fixes tests, we can make the 1.19 change later)

Comment (kaovilai) [reply build/ci-Dockerfile:2]: do you want to use 1.19 tag? or floating latest

**Review by sseago — APPROVED** [review]

File: build/ci-Dockerfile (hunk 1 above)
Comment (weshayutin) [reply build/ci-Dockerfile:2]: https://quay.io/repository/konveyor/builder?tab=tags looks like these were built 5 months ago.. so unclear to me if 1.19 is built.

Let's follow up in a PR.  I'll look at this tomorrow

Comment (kaovilai) [reply build/ci-Dockerfile:2]:
```
❯ podman run --rm quay.io/konveyor/builder:latest go version
WARNING: image platform ({amd64 linux  [] }) does not match the expected platform ({arm64 linux  [] })
go version go1.18.4 linux/amd64

~/oadp-operator oadp-1.1-build-ci-dockerfile-chmod
❯ podman run --rm quay.io/konveyor/builder:v1.19 go version
Trying to pull quay.io/konveyor/builder:v1.19...
Error: initializing source docker://quay.io/konveyor/builder:v1.19: reading manifest v1.19 in quay.io/konveyor/builder: manifest unknown: manifest unknown
```
no 1.19 images yet

**Pull request merged**